cves/kernel/CVE-2016-4998.yml

```diff
@@ -356,8 +309,8 @@ forgotten_check:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: true
+  note: there was a missing check to see that the data being accessed was within the active blob
```

Comment:
Missing period and first word in sentence should be capitalized. Also "the data being accessed was within the active blob" feels a little too jargony for readers without further explanation.